client/templates/layouts/header/languageSelection/languageSelection.js

```diff
@@ -0,0 +1,7 @@
+Template._header.events({
+    'change .language-selection': function(event) {
+        var language = $(event.target).val();
```

Comment:
Add a brief comment describing the code inside the change event. This way, a person can simply read the comment to understand what the code does.